### PR TITLE
use rubocop for linting. ran safe autocorrect

### PR DESCRIPTION
**Pull request by jdoconnor**
if this is of interest, I found a lot of ruby style convention updates. 

I intentionally created a config with violations that appeared to be intentional as to not rewrite developer intention.

### CODE REVIEW

**Comment by jdoconnor** [comment]
@camertron

File: lib/kuby/environment.rb [hunk 1]
```diff
@@ -4,19 +4,19 @@ class Environment
     attr_reader :name, :definition
     attr_accessor :configured
 
-    alias_method :configured?, :configured
+    alias configured? configured
```
Comment (camertron) [inline lib/kuby/environment.rb:7]: Let's keep `alias_method`, it's a lot more flexible than `alias`, can be overridden, etc.

File: lib/kuby/kubernetes/deployer.rb [hunk 2]
```diff
@@ -108,7 +108,7 @@ def restart_rails_deployment_if_necessary
               'deployment', namespace.metadata.name, deployment_name
             )
 
-            deployed_image = deployment.dig(*%w(spec template spec containers), 0, 'image')
+            deployed_image = deployment.dig('spec', 'template', 'spec', 'containers', 0, 'image')
```
Comment (camertron) [inline lib/kuby/kubernetes/deployer.rb:111]: Huh. Why does rubocop like `%w(a b c)` but not `dig(*%w(a b c))`??

File: lib/kuby/plugins/nginx_ingress.rb [hunk 3]
```diff
@@ -57,7 +57,7 @@ def already_deployed?
         kubernetes_cli.get_object('Service', 'ingress-nginx', 'ingress-nginx')
         true
       rescue KubernetesCLI::GetResourceError
-        return false
+        false
```
Comment (camertron) [inline lib/kuby/plugins/nginx_ingress.rb:60]: Are you sure these are equivalent?

**Review by camertron — CHANGES_REQUESTED** [review]
Thanks for doing this :) Def not a fan of all the things rubocop does, but this is a great start.

File: lib/kuby.rb [hunk 4]
```diff
@@ -39,9 +39,7 @@ class << self
     def load!(config_file = nil)
       config_file ||= ENV['KUBY_CONFIG'] || File.join('.', 'kuby.rb')
 
-      unless File.exist?(config_file)
-        raise MissingConfigError, "couldn't find Kuby config file at #{config_file}"
-      end
+      raise MissingConfigError, "couldn't find Kuby config file at #{config_file}" unless File.exist?(config_file)
```
[the remark this answers is not in the document]
Comment (jdoconnor) [reply lib/kuby.rb:42]: should be. I'll take a look

File: lib/kuby/plugins/nginx_ingress.rb (hunk 3 above)
Comment (jdoconnor) [reply lib/kuby/plugins/nginx_ingress.rb:60]: should be. that's the end of a method

**Comment by jdoconnor** [comment]
out of date. closing

**Pull request closed without merging**